recipe/meta.yaml

```
@@ -22,46 +22,59 @@ outputs:
       skip: true  # [py<39]
       ignore_run_exports_from:
         - zlib
+      script_env:
+        # Workaround an upstream conda-build issue w/pip & `outputs` by setting env vars manually.
+        # Drop once conda-build 24.7.0 is released.
+        # xref: https://github.com/conda/conda-build/issues/3993
+        - PIP_NO_BUILD_ISOLATION=False
+        - PIP_NO_DEPENDENCIES=True
+        - PIP_IGNORE_INSTALLED=True
+        - PIP_CACHE_DIR=pip_cache
+        - PIP_NO_INDEX=True
     requirements:
       build:
         - python                                 # [build_platform != target_platform]
         - cross-python_{{ target_platform }}     # [build_platform != target_platform]
         - numpy                                  # [build_platform != target_platform]
         - pybind11 >=2.6                         # [build_platform != target_platform]
-        - pkg-config  # [not win]
+        - {{ stdlib('c') }}
         - {{ compiler('c') }}
         - {{ compiler('cxx') }}
-        - libtool  # [unix]
+        - cmake                                  # [win]
+        - pkg-config                             # [unix]
```

Looks like you were right, we need `pkg-config` on Windows. Error [from CI]( https://dev.azure.com/conda-forge/feedstock-builds/_build/results?buildId=963561&view=logs&j=7df5a329-fcdb-5857-5720-a5dae94c0641&t=67fc0302-7ecf-51cb-2fd0-391a1258793e&l=888 )

```
extern\meson.build:11:17: ERROR: Dependency lookup for freetype2 with method 'pkgconfig' failed: Pkg-config for machine host machine not found. Giving up.
```

It appears that `freetype` does have the `.pc` file in the Windows package (checked locally on my Mac). So this should be doable

```
$ ls freetype-2.12.1-hdaf720e_2/Library/lib/pkgconfig 
freetype2.pc
```

Let's give it a try

```suggestion
        - cmake                                  # [win]
        - pkg-config
```

---

Hmm...still seeing this error. So there is likely more needed here